neard/res/genesis_config.json

```diff
@@ -224,4 +224,4 @@
   "protocol_treasury_account": "test.near",
   "fishermen_threshold": "10000000000000000000000000",
   "records": []
-}
\ No newline at end of file
+}
```

Comment:
should we revert this?

Reply:
seem, I either did it in some other diff, or it magically disappeared.